docs/usage.rst

```diff
@@ -89,14 +89,14 @@ datasets, you can use **--output-value** command-line option:
 Listing available data files
 ----------------------------
 
-**HTTP protocol**
-
 In order to get a list of data files belonging to a record, please use
-the **get-file-locations** command:
+the **get-file-locations** or **download-files --dry-run** command:
+
+**HTTP protocol**
 
 .. code-block:: console
 
-    $ cernopendata-client get-file-locations --recid 5500
+    $ cernopendata-client download-files --recid 5500 --dry-run
```

Comment:
Please revert back to 'get-file-locations`... We `dry-run` can be best advertised in its own `download-files` section.

Reply:
Done 👍